flume-ng-doc/sphinx/FlumeUserGuide.rst

```diff
@@ -1182,7 +1182,6 @@ headers.<filegroupName>.<headerKey> --                             Header value
 byteOffsetHeader                    false                          Whether to add the byte offset of a tailed line to a header called 'byteoffset'.
 skipToEnd                           false                          Whether to skip the position to EOF in the case of files not written on the position file.
 idleTimeout                         120000                         Time (ms) to close inactive files. If the closed file is appended new lines to, this source will automatically re-open it.
-writePosInterval                    3000                           Interval time (ms) to write the last position of each file on the position file.
```

Comment:
Do we have to remove this?

Reply:
Perhaps it's better to keep it to be compatible with other channels. I'll add it back and modify the user guide.